services/elections.request.service.ts

```diff
@@ -0,0 +1,340 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { RequestService } from './request.service';
+import { Observable } from 'rxjs/internal/Observable';
+import { map } from 'rxjs/internal/operators/map';
+import { Election, Position, Candidate, Vote, Ballot, BallotPOST } from 'src/shared-ng/interfaces/elections';
+
+@Injectable()
+export class ElectionsRequestService extends RequestService {
+  baseURL = 'elections';
+
+  constructor(http: HttpClient) {
+    super(http);
+  }
+  ////////////////////
+  // Elections
+  ///////////////////
+  /**
+   * Lists elections
+   *
+   * https://docs.aswwu.com/?url=https://raw.githubusercontent.com/ASWWU-Web/python_server/develop/docs/elections.yml#/election/get_election
+   * @param queryParams
```

Comment:
when the query parameters are being used to filter a list we should name the parameter something like filterParams and also describe in the docstring that the object is used for filtering results
```suggestion
   * @param filterParams an object with properties used to filter the results
```